gym/wrappers/frame_skipping.py

```diff
@@ -0,0 +1,34 @@
+import gym
+
+__all__ = ['SkipWrapper']
+
+def SkipWrapper(repeat_count):
+    class SkipWrapper(gym.Wrapper):
+        """
+            Generic common frame skipping wrapper
+            Will perform action for `x` additional steps
+        """
+        def __init__(self, env):
+            super(SkipWrapper, self).__init__(env)
+            self.repeat_count = repeat_count
+            self.stepcount = 0
+
+        def _step(self, action):
+            done = False
+            total_reward = 0
+            current_step = 0
+            while current_step < (self.repeat_count + 1) and not done:
+                self.stepcount += 1
+                obs, reward, done, info = self.env.step(action)
+                if 'skip.stepcount' not in info:
```

Comment:
Why would anyone nest these?

If we don't have a use-case, I'd rather raise an error.